it/mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -1,26 +1,26 @@
-<!ENTITY aboutSupport.appBasicsAppBuildID "Application Build ID">
+<!ENTITY aboutSupport.appBasicsAppBuildID "ID Build applicazione">
 
-<!ENTITY aboutSupport.accountsTitle "Mail and News Accounts">
-<!ENTITY aboutSupport.showPrivateData.mainText "Include account names">
-<!ENTITY aboutSupport.showPrivateData.explanationText "(possibly identifying information)">
+<!ENTITY aboutSupport.accountsTitle "Account E-mail e News">
+<!ENTITY aboutSupport.showPrivateData.mainText "Includi nomi account">
+<!ENTITY aboutSupport.showPrivateData.explanationText "(informazioni identificabili)">
```

Comment:
INCOMPLETE: The word "possibly" is not rendered in the translation. I would suggest to modify this entry with "informazioni potenzialmente identificabili".